framework/OpenMod.Core/Localization/ProxyStringLocalizer.cs

```diff
@@ -21,13 +21,6 @@ public IEnumerable<LocalizedString> GetAllStrings(bool includeParentCultures)
             return m_StringLocalizer.GetAllStrings(includeParentCultures);
         }
 
-#pragma warning disable 618 // disable obsolete warning
-        public IStringLocalizer WithCulture(CultureInfo culture)
-        {
-            return m_StringLocalizer.WithCulture(culture);
-        }
-#pragma warning restore 618
-
```

Comment:
Maybe instead of removing, mark it as obsolete and just return IStringLocalizer

Reply:
done

Reply:
It should be removed, it's now unused in OpenMod. Obsolete will not work here, due to using `IStringLocalization` interface.

Reply:
Done